django_feature_flip/models.py

```diff
@@ -2,7 +2,11 @@
 
 
 class Feature(models.Model):
-    name = models.CharField(max_length=40, unique=True)
+    name = models.CharField(max_length=40, unique=True, null=False)
+    totally_enabled = models.BooleanField(default=False, null=False)
+
+    def enabled(self, actor):
+        return self.totally_enabled
```

Comment:
Will the `actor` param be used in the future?

Reply:
Yesss, It is being used in #11